README.md

```diff
@@ -426,3 +426,62 @@ If the Peoplefinder is to be successful, profiles need to be populated and maint
 ###Support
 
 A support email address is set as SUPPORT_EMAIL.
+
+## Keeping secrets and sensitive information secure
+
+### git-secrets
+
+To prevent the commitment of secrets and credentials into git repositories we use awslabs / git-secrets (https://github.com/awslabs/git-secrets)
+
+For MacOS, git-secrets can be install via Homebrew.  From the terminal run the following:
+
+    $ brew install git-secrets
```

Comment:
As this is a markdown file might be good to use the back ticks and make them code formatted `like this`